docs/operators/xmlParserOperator.md

```diff
@@ -0,0 +1,83 @@
+Xml Parser
+=============
+
+## Operator Objective
+The XmlParser operator parses XML records and constructs POJOs ("Plain Old Java Objects") from them. The operator also emits each record as a DOM Document if the relevant output port is connected. User can also provide a XSD (XML Schema Definition) to validate incoming XML records. Valid records will be emitted as POJOs / DOM Document while invalid ones are emitted on error port with an error message if the error port is connected.
+
+XmlParser is **idempotent**, **fault-tolerant** and **statically/dynamically partitionable**.
+
+## Class Diagram
+![](images/xmlParser/XmlParser.png)
+## Operator Information
+1. Operator location:**_malhar-library_**
```

Comment:
Please add a space after the colon to separate the label from the value, here and in all subsequent similar items.